libkpod/remove.go

```diff
@@ -0,0 +1,62 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/sirupsen/logrus"
+)
+
+// RemoveContainers removes the given containers. If the container is running, the container
+// should be force removed.
+func (c *ContainerServer) RemoveContainers(containers []string, force bool) string {
+	var ret string
+	for _, container := range containers {
+		id, err := c.remove(container, force)
+		if err != nil {
+			ret = append(ret, err.Error(), "\n")
```

Comment:
Docker prints out either the container id or an error message for each container.  I wasn't sure how to handle that with the way we report errors through the cli package.  Take a look at what I changed it to in my most recent push and see if that's any better